tests1/myfile1.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+echo $_POST[ 'phpcs should catch this problem' ];
```

Comment:
:no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$_POST['phpcs should catch this problem']' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).